src/pages/dataTableUtilities/actions/tableActions.js

```diff
@@ -161,6 +161,19 @@ export const addItem = (item, addedItemType, route) => (dispatch, getState) => {
   }
 };
 
+export const addFridge = route => dispatch => {
+  UIDatabase.write(() => {
+    const fridgeLocationType = UIDatabase.objects('LocationType').filtered(
+      'description == $0',
+      'fridge'
+    )[0];
```

Comment:
this is not great, but I am thinking to add a `LocationType` to desktop update/upgrade/startup code which ensures there is always a `LocationType` with a type.. I was thinking of using something more like `@sussol/fridge` - similar to `inventory_adjustment` name record

Reply:
yes, was wondering the same thing, what happens when there isn't a fridge `LocationType`?
wouldn't there be a `Location` with an undefined `LocationType`? 
I think better to check for a `LocationType` - Toast if none exist and add if some do.

Reply:
I am thinking actually that using a `Form` to create `Locations` might be better. That way can just have a drop down for selecting a location type. But actually, maybe just adding a location type cell and instead calling this 'dataset' 'locations'? And just leave it up to the user to make sure they have selected the right location type..

Reply:
yeah, sounds interesting. just keep it flexible - the user picks the location type, and in this case chooses a 'fridge' (hopefully). +1 from me.